packages/frontend/src/server/database.ts

```diff
@@ -21,3 +19,30 @@ function createThrowingProxy() {
     },
   })
 }
+
+function createConnectionTag() {
+  const base = 'l2beat-frontend'
+
+  return `${base}-${env.NODE_ENV}`
```

Comment:
isn't it better to use vercel's preview id somehow?

Reply:
https://github.com/l2beat/l2beat/pull/5259/commits/2c090f79bbf1c70814c841abeadee0d2e8e99759